.github/composite-actions/build-extensions/action.yml

```diff
@@ -8,16 +8,23 @@ inputs:
     description: 'Extension Branch name which needs to checkout first'
     required: no
     default: 'not_specified'
+  is_extension_version_latest: 
+    description: 'Whether the Extension Version that needs to be build is latest'
+    required: no
+    default: 'false'
 
 runs:
   using: "composite"
   steps:
-    - if: always() && (inputs.extension_branch != 'not_specified')
+    - if: always() && (inputs.extension_branch != 'not_specified') && (inputs.is_extension_version_latest == 'false')
```

Comment:
can we combine latest and 'not specified' into one string say 'default'. So 14.latest will pass as brach 'default' and we use it pick pr branch

Reply:
done